CHANGELOG.md

```diff
@@ -1,8 +1,30 @@
-## 0.6.0 (Unreleased)
-## 0.5.0 (April 22, 2020)
+## 0.5.1 (Unreleased)
+
+[Full Changelog](https://github.com/terraform-providers/terraform-provider-mongodbatlas/compare/v0.5.0...HEAD)
+
+**Implemented enhancements:**
+
+- Support new private and privateSrv connection strings [\#183](https://github.com/terraform-providers/terraform-provider-mongodbatlas/issues/183)
+
+**Closed issues:**
+
+- Alert configuration roles array should not be required [\#201](https://github.com/terraform-providers/terraform-provider-mongodbatlas/issues/201)
+- Can't get PrivateLink-aware mongodb+srv address when using privatelink [\#147](https://github.com/terraform-providers/terraform-provider-mongodbatlas/issues/147)
```

Comment:
This will also close issue 200 so should be in this list too. 
 terraform plan and apply fails after upgrading this module to 0.5.0 [\#200](https://github.com/terraform-providers/terraform-provider-mongodbatlas/issues/200)

Reply:
thanks let me add it :)